ReadMe.md

```diff
@@ -188,6 +188,9 @@ Virtual machine | 仮想マシン | かそうマシン |
 Server layout, architecture | サーバ構成 | サーバこうせい |
 Static IP | 固定IP | こていIP |
 Load | 負荷 | ふか |
+Scalability | 拡張性 | かくちょうせい |
+Abailability | 可用性 | かようせい |
+Alcohol | お酒 | おさけ |
```

Comment:
We try to limit this list to IT words only. I like alcohol, but maybe not in this list. Could you remove it?